src_py/apiServer/stats.py

```diff
@@ -80,6 +82,12 @@ def get_loss_ts(self , plot : bool = False , saveToFile : bool = False): # Todo
         df = pd.DataFrame(loss_dict)
         self.loss_ts_pd = df
         #print(df)
+        
+        if plot:
+            sns.lineplot(data=df)
+            plt.xlabel('Batch Num.')
```

Comment:
@GuyPerets106 
interesting that it allows you to mix sns and plt

Reply:
Seaborn is based on matplotlib , it just add to the visuals and capabilities 😄